runtime/src/accounts_db.rs

```diff
@@ -4246,11 +4246,19 @@ impl AccountsDb {
         ancestors: &Ancestors,
     ) -> (Hash, u64) {
         if !use_index {
+            let mut time = Measure::start("collect");
```

Comment:
maybe "snapshot_collect" instead of "collect" which is too generic?

Reply:
this string doesn't get used anywhere. If you print 'time', it will use the string, but I want numbers out, so I call as_us().

Reply:
okay